internal/criocli/criocli.go

```diff
@@ -638,7 +638,7 @@ func getCrioFlags(defConf *libconfig.Config) []cli.Flag {
 		&cli.StringSliceFlag{
 			Name:    "registry",
 			Value:   cli.NewStringSlice(defConf.Registries...),
-			Usage:   "Registry to be prepended when pulling unqualified images. Can be specified multiple times.",
+			Usage:   "Registry to be prepended when pulling unqualified images. Can be specified multiple times. This option is deprecated, and it has no effect.",
```

Comment:
Using the same format as the other deprecated command-line switch.

```suggestion
			Usage:   "Registry to be prepended when pulling unqualified images. Can be specified multiple times. DEPRECATED: This option will be removed in the future, and it has no effect.",
```

Reply:
... or maybe not. Other older deprecations didn't follow this sort of loud way.